admin/etc/adminhtml/system.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Pronko Consulting (https://www.pronkoconsulting.com)
+ * See LICENSE for the license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <section id="payment">
+            <group id="liqpay_section" sortOrder="6" showInDefault="0" showInWebsite="0" showInStore="0">
+                <group id="liqpay" translate="label" type="text" sortOrder="100" showInDefault="1" showInWebsite="1" showInStore="1">
+                    <label>LiqPay</label>
+                    <frontend_model>Magento\Paypal\Block\Adminhtml\System\Config\Fieldset\Payment</frontend_model>
```

Comment:
We shouldn't be based on the Magento_Paypal module, it is better to have a simple version of a config without dependencies and then later we can create our own fieldset class.